src/olympia/accounts/utils.py

```diff
@@ -18,28 +18,6 @@
 from .tasks import clear_sessions_event, delete_user_event, primary_email_change_event
 
 
-def fxa_config(request):
```

Comment:
Just curious, with the removal of this, what was it ever used for?

Reply:
It was used in the `fxa_config` helper which was called from the base template to add a `data-` attribute containing that info. That would presumably have been useful a long time ago with the legacy site.